morello/morello.md

```diff
@@ -1,23 +1,9 @@
 # Morello Supplement to the Arm C Language Extensions
 
-::: version
 01alpha
-:::
-
-::: issued
 Date of Issue: 02 July 2021
-:::
-
-::: logo
 ![image](Arm_logo_blue_RGB.svg)
```

Comment:
Each of these need to be followed by an empty line otherwise they will show up in the same paragraph. As of now they look in a single line as here:
<img width="846" alt="Screenshot 2021-09-30 at 18 04 03" src="https://user-images.githubusercontent.com/25690309/135499641-fba20d1c-2c14-4ebe-ac2e-d9d148ab7dc6.png">